File: coin_toss/coin_toss.js
function coin_toss(n=10000){
    count = {"H": 0, "T": 0}
    for (let i=0;i<n;i++){
        const num = Math.floor(Math.random() * 2);
        if (num === 0){
            count['H'] += 1
        } else {
            count['T'] += 1
        }
    }
    return count
}

File: coin_toss/coin_toss.swift
func coin_toss(n: Int = 10000) -> [String: Int] {
    var count = ["H": 0, "T": 0]
    for _ in 1...n {
        let toss = Bool.random() ? "H" : "T"
        count[toss]! += 1
    }
    return count
}